app/models/news/collection.rb

```diff
@@ -29,5 +29,9 @@ def matches_param?(collection_id)
         false
       end
     end
+
+    def stop_press?
```

Comment:
You could use the enum functionality here? Can be convenient for this kind of thing